### ISSUE TITLE
Error when compiling on Mac

### ISSUE THREAD

**Issue by devbanana** [matatk/agrip#57]
Just found this project today and been trying to build. I get through all the steps fine, but when I run python3 build.py, I get an error like this:

Error: failed to compile gamecode file: progs.src

Been trying to find any solution but can't seem to figure it out.

Here's the complete output:

```
Building 2019.0 : Respawn
Preparing downloaded assets dir
Preparing converted (HTML) manual dir
Compiling zqcc
Compiling zquake
Compiling gamecode
Error running: ('/Users/brandonolivares/Downloads/agrip-2019.0-beta1-mac/audioquake/zq-repo/zqcc/zqcc', '-progs', 'progs.src') - trying again, with full output...
Progs-Sourcefile:  progs.src
Additional reserved #define: _ZQCC
Outputfile:       ./qwprogs.dat
Compiling ../qw/defs.qc
Compiling ../qw/defs_ext.qc
Compiling agrip_defs.qc
Additional #define: AGRIP
Compiling ../qw/subs.qc
Compiling ../qw/combat.qc
Compiling ../qw/items.qc
Compiling epi.qc
Compiling se.qc
Compiling misc.qc
Compiling nav.qc
Compiling extnav.qc
Compiling d5k.qc
Compiling esr.qc
Compiling marker.qc
Compiling hooks.qc
Compiling ../qw/weapons.qc
Compiling ../qw/world.qc
Compiling ../qw/client.qc
Compiling ../qw/spectate.qc
Compiling ../qw/player.qc
Compiling ../qw/doors.qc
Compiling ../qw/buttons.qc
Compiling ../qw/triggers.qc
Compiling ../qw/plats.qc
Compiling ../qw/misc.qc
Compiling ../qw/server.qc
Compiling ../qw/commands.qc
Compiling ../qwsp/sp_defs.qc
Additional #define: QWSP
Compiling tutor_support.qc
Compiling tutor.qc
Error: failed to compile gamecode file: progs.src

```

**Comment by matatk** [comment]
Hi @devbanana, thanks for your bug report. It looks like you're using an outdated version of the code. The [current release available for download is 2020.0-beta1](https://github.com/matatk/agrip/releases/tag/2020.0-beta1), and you shouldn't need to build that to try it out. However, if you do want to build it locally, you can get the latest source code either by downloading the source code as a ZIP file from that same link, or using git to grab the latest code (i.e. `git clone https://github.com/matatk/agrip/`). Hopefully with the latest code you will find the error has gone.

**Comment by matatk** [comment]
Can't reproduce, so closing this for now; feel free to re-open if it's still a problem. Thanks again for your bug report.
